### ISSUE TITLE
Scheduler tutorial

### ISSUE THREAD

**Issue by JunCTionS** [GLORIA-project/users-hub-interface#37]
A nice little tutorial for the scheduler might be nice. 

In fact a dropdown menu with all the tutorials would be ideal, and even better if not only in video format.

Another user wrote to me (maybe he got my name from an outreach talk) regarding the lack of information on filters and how to build a color image.

**Comment by marsfan** [comment]
I second this. Gloria seems to have the power to become a very powerful system, but the lack of information on how to use it means that only really knowledgeable people can use it.

**Comment by manu9035** [comment]
estoy de acuerdo ya que proyecto gloria es una herramienta muy buena para ayudarnos  a los aficionados noveles en este mundillo tan apasionante.

**Comment by marsfan** [comment]
It seems pretty easy,but many if the options on the telescope (filter, etc) are very hard to understand unless I already use telescopes professionally

**Comment by FrancescoAdi** [comment]
Dear All,
I just started this week to try Gloria and I agree with the fact that must be provided a full user manual for every telescope, or better, a tutorial the likes that can be found on other Citizen science sites on the net.
This way every one can try without frustration and improve this nice idea with everyone personal contribute.
Otherwise people, instead of being attracted from astronomy, can be once and for all scared and leave the subject unsatisfied.
